src/cryptonote_basic/cryptonote_basic.h

```diff
@@ -259,7 +259,7 @@ namespace cryptonote
             ar.tag("rctsig_prunable");
             ar.begin_object();
             r = rct_signatures.p.serialize_rctsig_prunable(ar, rct_signatures.type, vin.size(), vout.size(),
-                vin[0].type() == typeid(txin_to_key) ? boost::get<txin_to_key>(vin[0]).key_offsets.size() - 1 : 0);
```

Comment:
Line 251 already checks for vin.empty(). Can it change in the meantime?

Reply:
I don't think so.